tests/cases/standalone/common/create/create.sql

```diff
@@ -57,3 +57,17 @@ CREATE TABLE neg_default_value(i INT DEFAULT -1024, ts TIMESTAMP TIME INDEX);
 desc TABLE neg_default_value;
 
 DROP TABLE neg_default_value;
+
+CREATE TABLE test_like_1 (i INTEGER, j TIMESTAMP TIME INDEX);
```

Comment:
Maybe make this case a bit more complicated, including default value, primary key, big letter identifier etc. The previous one could not recognize column constrain.

Reply:
+1

Reply:
> Maybe make this case a bit more complicated, including default value, primary key, uppercase identifier etc. The previous one could not recognize column constraints.

whether secondary indexes are required, it seems that there is no support for the moment